applications/services/gui/modules/file_browser_worker.c

```diff
@@ -87,7 +88,8 @@ static bool browser_filter_by_name(BrowserWorker* browser, FuriString* name, boo
     if(is_folder) {
         // Skip assets folders (if enabled)
         if(browser->skip_assets) {
-            return ((furi_string_cmp_str(name, ASSETS_DIR) == 0) ? (false) : (true));
+            return ((furi_string_cmp_str(name, ASSETS_DIR) == 0) ? (false) : (true)) &&
+                   ((furi_string_cmp_str(name, BADUSB_LAYOUTS_DIR) == 0) ? (false) : (true));
```

Comment:
This was done because in BadUSB app it shows that folder because it located in same folder as badusb scripts, it can be opened and will be empty because we looking for .txt files not .kl ones, hiding it in filebrowser seems correct solution, we still can access it in "Archive" app, in "Browser" tab where assets is not hidden

So, is there a reason to show this "empty" folder when we looking for .txt scripts?

Reply:
Sorry for delay, file browser module has only one reserved name for hidden folder - "assets". We don't plan to extend these names to keep file browser independent from other apps features. Also is a bug: if you have selected layout file from outside of layouts folder, you can't open layouts folder anymore, because it's hidden in file browser.

So yes, now it's better to keep layouts folder visible

Reply:
Maybe better to rename layouts folder to assets so it will not be different from other apps?
What do you think

Reply:
Assets folders contain files that are used by app directly and should be never shown in file browser. Anyway, I don't see any issue in showing layouts folder everywhere

Reply:
But we're using file browser module on layout select page. If "layouts" folder is in file browser hide-list, you just can't open in a file browser that starts on a layout select page. You can disable skip_assets option for this page, but it will lead to showing "assets" folders also

Reply:
> But we're using file browser module on layout select page. If "layouts" folder is in file browser hide-list, you just can't open in a file browser that starts on a layout select page. You can disable skip_assets option for this page, but it will lead to showing "assets" folders also

No, Actually it works correctly, but if you go to previous folders with back button then yes, you can't open that folder again, need to close filebrowser dialog and open it again from config page
Actually when it was implemented there was a bit different navigation, back button was closing dialog instead of going back on file tree
I need to think what can be done here, and ill come back with changes today

Reply:
Ok, we're planning to merge this pr soon with small fixes from me, you can open another one later

Reply:
I already prepared some 
changes, Ill push them in next 20 mins